python/tvm/topi/vision/ssd/multibox.py

```diff
@@ -230,9 +242,9 @@ def hybrid_multibox_transform_loc(cls_prob, loc_pred, anchor, clip, threshold, v
                 out_loc[i, valid_count[i], 0] = cls_id - 1.0
                 out_loc[i, valid_count[i], 1] = score
                 for l in range(4):
-                    box_coord[l] = anchor[0, j, l]
-                    pred_coord[l] = loc_pred[i, j * 4 + l]
-                out_coord = _hybridy_transform_loc(box_coord, pred_coord, variances, clip)
+                    box_coord[i, l] = anchor[0, j, l]
```

Comment:
It seems `box_coord` is the same in all threads. How about passing `anchor`, `j`, and `l` to `_hybridy_transform_loc` instead?

Reply:
good idea~ updated as suggested.